yb-voyager/cmd/analyzeSchema.go

```diff
@@ -625,6 +598,32 @@ func reportUnsupportedIndexesOnComplexDatatypes(createIndexNode *pg_query.Node_I
 	}
 }
 
+var unsupportedIndexMethods = []string{
+	"gist",
+	"brin",
+	"spgist",
+}
+
+func reportIndexMethods(createIndexNode *pg_query.Node_IndexStmt, sqlStmtInfo sqlInfo, fpath string) {
+	indexMethod := createIndexNode.IndexStmt.AccessMethod
+	
+	if !slices.Contains(unsupportedIndexMethods, indexMethod) {
+		return
+	}
+
+	indexName := createIndexNode.IndexStmt.GetIdxname()
+	relName := createIndexNode.IndexStmt.GetRelation()
+	schemaName := relName.GetSchemaname()
+	tableName := relName.GetRelname()
+	fullyQualifiedName := lo.Ternary(schemaName != "", schemaName+"."+tableName, tableName)
+	displayObjName := fmt.Sprintf("%s ON %s", indexName, fullyQualifiedName)
+
+	summaryMap["INDEX"].invalidCount[displayObjName] = true
+	
+	reportCase(fpath, fmt.Sprintf(INDEX_METHOD_ISSUE_REASON, strings.ToUpper(indexMethod)),
+		"https://github.com/YugaByte/yugabyte-db/issues/1337", "", "INDEX", displayObjName, sqlStmtInfo.formattedStmt, UNSUPPORTED_FEATURES, GIST_INDEX_DOC_LINK)
```

Comment:
GIST_INDEX_DOC_LINK : this should be changed?

Reply:
I was thinking of updating the same issue on docs to include all the other ones. will see if the link changes.